### DISCUSSION TITLE
Show info somewhere in GUI if Ollama is accelerated by GPU or not

### DISCUSSION THREAD

**Discussion by DocMAX** [open-webui/open-webui#2558] [General]
Would be great, thanks!

**Comment by Kamil-Jasinski** [comment]
Cool idea but I would expand it a bit to show current RAM/CPU/GPU usage.

If you currently want to check if Ollama is supported by the GPU you can open the task manager to the performance tab and check it yourself.

**Comment by DocMAX** [comment]
there is no performance tab? or what do you mean?

**Reply by flefevre** [reply]
Remember that open WebUI is now multi compatible with ollama and addionnaly with litellm and so access to more than 100' llm server, such as vllm

**Comment by silentoplayz** [comment]
Running `ollama ps` in the command line interface will tell you if a model is loaded into your GPU(s) or not, and if so, the percentage of the model that is loaded into the GPU(s).

**Reply by justinh-rahb** [reply]
Maybe we should implement this new API endpoint they added not long ago:
https://github.com/ollama/ollama/blob/main/docs/api.md#list-running-models

**Reply by gaby** [reply]
This should be shown in the Admin UI though.
